sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventData.java

```diff
@@ -162,9 +144,21 @@ public EventData(BinaryData body) {
      * @return A byte array representing the data.
      */
     public byte[] getBody() {
-        return body.toBytes();
+        final AmqpMessageBodyType type = amqpAnnotatedMessage.getBody().getBodyType();
+        switch (type) {
+            case DATA:
+                return amqpAnnotatedMessage.getBody().getFirstData();
+            case SEQUENCE:
```

Comment:
No logic for this case?

Reply:
I think this is fine. It'll drop to the case below since both of them are unsupported at the moment.